odata_renderer/src/test/java/com/sdl/odata/renderer/metadata/MetadataDocumentFunctionWriterTest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014 All Rights Reserved by the SDL Group.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.sdl.odata.renderer.metadata;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * <p>
+ * Tests for MetadataDocumentFunctionWriter.
+ * </p>
+ */
+public class MetadataDocumentFunctionWriterTest {
```

Comment:
Can we have some tests added ?

Reply:
@vinayknl The automated tests seem expect something different on my local machine and Travis seems to expect inconsistent result...

That is, in the expected value is (from test metadata.xml lines 140 & 141):
`...<Parameter Name="stringFunctionField" Type="String" Nullable="true"/>		
<Parameter Name="intFunctionField" Type="Int16" Nullable="true"/>...`

...but sometimes the writer produces
`...<Parameter Name="intFunctionField" Type="Int16" Nullable="true"/>
    <Parameter Name="stringFunctionField" Type="String" Nullable="true"/>...`
                 
Has anyone else wrestled with this, or am I doing something wrong?